Core/OfficeDevPnP.Core/AppModelExtensions/FieldAndContentTypeExtensions.cs

```diff
@@ -739,6 +739,10 @@ public static void AddFieldToContentTypeByName(this Web web, string contentTypeN
         public static void AddFieldToContentType(this Web web, ContentType contentType, Field field, bool required = false, bool hidden = false)
         {
             contentType.EnsureProperties(c => c.Id, c => c.FieldLinks, c => c.SchemaXml);
```

Comment:
Related to this fix. I’m using the latest code and I found that when I’m adding 2 or more fields to an existing content type the following line in Field and ContenttYpeExtensions.cs ( line 748) still returns the same id exception

  var flink = contentType.FieldLinks.FirstOrDefault(fld => fld.Id == field.Id);

The first field is added but the 2nd field can’t be added.

From: Sergei Sergeev [mailto:notifications@github.com]
Sent: 08 October 2015 09:53
To: OfficeDev/PnP-Sites-Core <PnP-Sites-Core@noreply.github.com>
Subject: Re: [PnP-Sites-Core] Making sure IDs are loaded for all FieldLinks in a ContentType with EnsureProperties (#93)


In Core/OfficeDevPnP.Core/AppModelExtensions/FieldAndContentTypeExtensions.cs<https://github.com/OfficeDev/PnP-Sites-Core/pull/93#discussion_r41489914>:

> @@ -739,6 +739,10 @@ public static void AddFieldToContentTypeByName(this Web web, string contentTypeN

>          public static void AddFieldToContentType(this Web web, ContentType contentType, Field field, bool required = false, bool hidden = false)

>          {

>              contentType.EnsureProperties(c => c.Id, c => c.FieldLinks, c => c.SchemaXml);

@SchauDK<https://github.com/SchauDK>, I will look today's evening (GMT +3) :)

—
Reply to this email directly or view it on GitHub<https://github.com/OfficeDev/PnP-Sites-Core/pull/93/files#r41489914>.